charts/kyverno/README.md

```diff
@@ -172,6 +171,13 @@ It contains an array of string templates that are passed through the `tpl` Helm
 
 Please consult the [values.yaml](./values.yaml) file before overriding `config.resourceFilters` and use the apropriate templates to build your desired exclusions list.
 
+## High availability
+
+Running highly available Kyverno is crucial as the Kubernetes api server will invoke Kyverno webhook for almost all resource operations in the cluster.
```

Comment:
Stating blindly "will invoke Kyverno webhook for almost all resource operations in the cluster" isn't quite true as this would depend on the number and type of policies a user has created. It's best to not make this statement.

Reply:
Done